jobs/route_registrar/spec

```diff
@@ -36,31 +37,52 @@ properties:
     description: (string, optional) By default, route_registrar will detect the IP of the VM and use it, in combination with port as the backend destination for each uri being registered. This property enables overriding the destination hostname or IP.
     example: 192.168.60.25
 
+  route_registrar.routing_api.api_url:
+    description: (optional, string) The routing API's URL. This is required to register any TCP routes.
+    default: http://routing-api.service.cf.internal:3000
+  route_registrar.routing_api.oauth_url:
+    description: (optional, string) The OAuth server's URL. This is required to register any TCP routes.
+    default: https://uaa.service.cf.internal:8443
+  route_registrar.routing_api.client_id:
+    description: (optional, string) An OAuth client ID for a client that is permitted to add new TCP routes. This is required to register any TCP routes.
+    default: routing_api_client
+  route_registrar.routing_api.client_secret:
+    description: (optional, string) The OAuth client secret for the above client. This is required to register any TCP routes.
+  route_registrar.routing_api.ca_certs:
+    description: (optional, array of strings) The certificate authority certificates for any APIs that the route registrar is communicating with over HTTPS, e.g., the OAuth server. This is required to register any TCP routes.
+  route_registrar.routing_api.skip_cert_validation:
```

Comment:
I don't think this is necessary. Notice that other jobs in routing-release respect a top-level, job agnostic property `skip_ssl_validation`